proto/buf.gen.yaml

```diff
@@ -9,23 +9,23 @@ managed:
     except:
```

Comment:
The comment on top of the file isn't needed anymore. Also, we don't need the template in the buf schema registry anymore, since we are generating the code ourselves with `buf.gen.yaml` (this is our "template").